src/app/pages/WelcomePage/welcomePage.tsx

```diff
@@ -0,0 +1,58 @@
+          >
+            <path
+              className={styles.icon}
+              d="M26.8008 35V36.6406C26.8008 37.5781 26.4753 38.3724 25.8242 39.0234C25.1732 39.6745 24.3789 40 23.4414 40H10.1602C9.22265 40 8.42838 39.6745 7.77734 39.0234C7.1263 38.3724 6.80078 37.5781 6.80078 36.6406V35C5.34245 35 3.87109 34.3359 2.38672 33.0078C0.902341 31.6797 0.160153 30.1302 0.160153 28.3594V13.3594C0.160153 11.5885 0.902341 10.026 2.38672 8.67188C3.87109 7.31771 5.34245 6.64062 6.80078 6.64062H8.4414V5C8.4414 3.59375 8.92317 2.40885 9.88672 1.44531C10.8503 0.481771 12.0352 0 13.4414 0H20.1602C21.5664 0 22.7513 0.481771 23.7148 1.44531C24.6784 2.40885 25.1602 3.59375 25.1602 5V6.64062H26.8008C28.6237 6.64062 30.1862 7.30469 31.4883 8.63281C32.7904 9.96094 33.4414 11.5365 33.4414 13.3594V28.3594C33.4414 30.1823 32.7904 31.7448 31.4883 33.0469C30.1862 34.349 28.6237 35 26.8008 35ZM10.1602 36.6406H23.4414V35H10.1602V36.6406ZM21.8008 5C21.8008 3.90625 21.2539 3.35938 20.1602 3.35938H13.4414C12.3477 3.35938 11.8008 3.90625 11.8008 5V6.64062H21.8008V5ZM30.1602 13.3594C30.1602 12.4219 29.8216 11.6276 29.1445 10.9766C28.4674 10.3255 27.6862 10 26.8008 10H6.80078C6.17578 10 5.45963 10.3516 4.65234 11.0547C3.84505 11.7578 3.4414 12.526 3.4414 13.3594V28.3594C3.4414 29.1927 3.84505 29.9479 4.65234 30.625C5.45963 31.3021 6.17578 31.6406 6.80078 31.6406H26.8008C27.6862 31.6406 28.4674 31.3151 29.1445 30.6641C29.8216 30.013 30.1602 29.2448 30.1602 28.3594V13.3594ZM20.1602 17.5C20.1602 18.2292 20.4076 18.8281 20.9023 19.2969C21.3971 19.7656 21.9831 20 22.6602 20C23.3372 20 23.9232 19.7526 24.418 19.2578C24.9128 18.763 25.1602 18.1771 25.1602 17.5C25.1602 16.8229 24.9128 16.237 24.418 15.7422C23.9232 15.2474 23.3372 15 22.6602 15C21.9831 15 21.3971 15.2344 20.9023 15.7031C20.4076 16.1719 20.1602 16.7708 20.1602 17.5ZM8.4414 17.5C8.4414 18.1771 8.6888 18.763 9.18359 19.2578C9.67838 19.7526 10.2643 20 10.9414 20C11.6185 20 12.2044 19.7656 12.6992 19.2969C13.194 18.8281 13.4414 18.2292 13.4414 17.5C13.4414 16.7708 13.194 16.1719 12.6992 15.7031C12.2044 15.2344 11.6185 15 10.9414 15C10.2643 15 9.67838 15.2474 9.18359 15.7422C8.6888 16.237 8.4414 16.8229 8.4414 17.5ZM16.8008 28.3594C14.5091 28.3594 12.556 28.0078 10.9414 27.3047C9.32682 26.6016 8.28515 25.9635 7.8164 25.3906C7.34765 24.8177 7.0612 24.349 6.95703 23.9844C6.54036 22.9948 6.82682 22.2656 7.8164 21.7969C8.80599 21.3802 9.53515 21.6667 10.0039 22.6562C10.0039 22.8125 10.1602 23.0469 10.4727 23.3594C10.7852 23.6719 11.5143 24.0234 12.6602 24.4141C13.806 24.8047 15.1862 25 16.8008 25C18.4154 25 19.7956 24.8047 20.9414 24.4141C22.0872 24.0234 22.8164 23.6458 23.1289 23.2812L23.5977 22.6562C24.0664 21.6667 24.7956 21.3802 25.7852 21.7969C26.7747 22.2656 27.0612 22.9948 26.6445 23.9844C26.5404 24.349 26.2539 24.8177 25.7852 25.3906C25.3164 25.9635 24.2747 26.6016 22.6602 27.3047C21.0456 28.0078 19.0924 28.3594 16.8008 28.3594Z"
+              fill="#8C0327"
+            />
+          </svg>
```

Comment:
I think these SVG's could be components. Like this:
https://github.com/marwinburesch/moviescout/blob/main/src/app/components/assets/BookmarkIcon.tsx

Reply:
Saved as Issue for later